README.md

```diff
@@ -2,8 +2,99 @@
 
 Repository containing pipeline for processing spinal cord injury (SCI) patients (both DCM and tSCI) using the [Spinal Cord Toolbox (SCT)](https://github.com/spinalcordtoolbox/spinalcordtoolbox).
 
-Steps:
+Pipeline steps:
 1. DICOM to nii (BIDS) conversion
 2. Processing (spinal cord and lesion segmentation, vertebral labeling)
 3. Quality control (QC) + manual compression level labeling
 4. Lesion metric computation
+
+## Table of contents
+* [1. Dependencies](#1-dependencies)
+* [2. Installation](#2-installation)
+  * [2.1 SCT Installation](#21-sct-installation)
+  * [2.2 dcm2niix Installation](#22-dcm2niix-installation)
+* [3. Processing pipeline](#3-processing-pipeline)
+
+## 1. Dependencies
+
+* [Spinal Cord Toolbox v6.4](https://github.com/spinalcordtoolbox/spinalcordtoolbox/releases/tag/6.4)
+* [dcm2niix >= v1.0.20220505](https://github.com/rordenlab/dcm2niix?tab=readme-ov-file#install)
+
+## 2. Installation
+
+> [!NOTE]
+> The installation process below is currently only supported on macOS.
+
+### 2.1 SCT Installation
+
+1. Open a new terminal:
+
+Press <kbd>command</kbd> + <kbd>space</kbd> and type `Terminal` and press <kbd>return/enter</kbd>.
+
+2. Run the following commands in the terminal (you can copy-paste the whole block):
+
+> [!NOTE]
+> The installation process will take a few minutes.
+
+```bash
+# Go to your home directory
+cd ~
+# Download SCT v6.4
+wget https://github.com/spinalcordtoolbox/spinalcordtoolbox/archive/refs/tags/6.4.zip
```

Comment:
Just a note: I use `wget` instead of `git clone` because Apple Developer Tools are needed for `git`.